pkg/operator/ceph/operator.go

```diff
@@ -197,6 +197,10 @@ func (o *Operator) startSystemDaemons(clusterSpec *cephv1.ClusterSpec) error {
 		return fmt.Errorf("invalid csi params: %v", err)
 	}
 
+	if _, err = csi.ValidateCSIVersion(o.operatorNamespace, o.context.Clientset, csi.DefaultCSIPluginImage, o.rookImage, o.securityAccount); err != nil {
```

Comment:
if the first value(bool) is discarded why `ValidateCSIVersion` return two values?